libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4179,6 +4187,8 @@ void GCS_MAVLINK::handle_command_int(const mavlink_message_t &msg)
     mavlink_msg_command_ack_send(chan, packet.command, result);
 
     AP::logger().Write_Command(packet, result);
+    gcs().send_text(MAV_SEVERITY_DEBUG, GCS::MessageOption::MAVLINK_COMMAND, false, "CmdI: %d (%g %g %g %g %" PRIu32 " %" PRIu32 " %g) Res: %d",
```

Comment:
I don't see the value of sending these out as status texts. There already is a MAVLink command ack for GCS's, and we already log these to dataflash logs, at that point this seems like this is just adding more of a dependency upon statustexts as an out of band ack. I don't think this is a step in the right direction.